internal/providersdkv2/resource_hvn_route.go

```diff
@@ -364,3 +404,50 @@ func resourceHVNRouteImport(ctx context.Context, d *schema.ResourceData, meta in
 
 	return []*schema.ResourceData{d}, nil
 }
+
+func getAzureConfig(propertyName string, d *schema.ResourceData) (*networkmodels.HashicorpCloudNetwork20200907AzureRoute, diag.Diagnostics) {
```

Comment:
Is there value in the property name being passed in? Will it ever have different values?

Reply:
Probably no value. I'll update 👍🏻